openerp/__init__.py

```diff
@@ -10,7 +10,11 @@
 import sys
 evented = False
 if sys.modules.get("gevent") is not None:
```

Comment:
Maybe replace this with a try/except ImportError pattern as well rather than go muck in sys.modules?

And considering lines 32/33 of this file, we should probably delete the imported modules afterwards to avoid re-exposing them (I assume that's the intent of lines 32/33 anyway)